### ISSUE TITLE
joshondesign.com/p/apps/aminogfx/build doesn't work

### ISSUE THREAD

**Issue by mrKrecik** [joshmarinacci/aminogfx#10]
Hey,

your repository https://joshondesign.com/p/apps/aminogfx/builds/ doesn't work, so installer cannot download aminonative. :/

**Comment by almilo** [comment]
Hi Josh,
It seems that this URL: https://joshondesign.com/p/apps/aminogfx/builds/aminonative-v0.0.2-node-v14-linux-arm.tar.gz is not working. Where can we find this file or how can we build it?
Thanks!!

**Comment by joshmarinacci** [comment]
Did you npm install aminogfx or aminogfx-gl?

**Comment by almilo** [comment]
I installed aminogfx-gl as explained here http://joshondesign.com/2015/02/27/aminorefactored and posted in this repo as it is also indicated in your blog post as 'feature requests'.
I am using a RPi 2 with Raspbian Wheezy.
Thanks!

**Comment by joshmarinacci** [comment]
so.... fun story.  After some investigation I've found that the directory containing the binary builds is gone. I don't know what happened to it. So I tried to rebuild on my Mac and it won't work. It seems that bit-rot has set it and Amino won't run on Node 0.12 due to native API changes from Node 0.10.  Given that the world has split and merged node into the forthcoming Node 4, it's time to update the code. This means I've got a lot of work to do switching to NAN so that native v8 changes won't break in the future. I'd been meaning to do this for a while but put it off. I've started the work and it should be a few more days.
